components/a11y/tests/unit/accessibility-helpers.spec.js

```diff
@@ -1,3 +1,24 @@
-test('TODO replace this', () => {
-	expect(true).toBe(true);
+import { GEL } from '@westpac/core';
+import { render } from '@testing-library/react';
+import { VisuallyHidden, SkipLink } from '@westpac/a11y';
+import wbc from '@westpac/wbc';
+
+// Component specific tests
+describe('Accessbility helper component', () => {
```

Comment:
Accessibility

Reply:
@wpalombini done! The other files have been changed because there were `console.errors` popping up from the render test. 
e.g. `Fork.name` was typed as required, but it is not since you can see that from the examples folder they are using without using that prop. So there were many inconsistencies that was showing with those tests that have been included. So I've changed them in order to fix those inconsistencies.